core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -125,13 +126,14 @@ var TopicsAndSkillsDashboardPage = function() {
     });
   };
 
-  this.createTopicWithTitle = function(title) {
+  this.createTopicWithTitle = function(title, shortName) {
```

Comment:
Use consistent terminology throughout the codebase -- either shortName or abbreviatedName, but not both.

Reply:
Done.